settings/global.mdx

```diff
@@ -375,7 +375,7 @@ Example:
 
 ### Redirects
 
-<ResponseField name="redirects" type="Redirect[]" required>
+<ResponseField name="redirects" type="Redirect[]">
```

Comment:
this shouldnt have been required i think lol

Reply:
lol i agree